tools/gulp/packaging/rollup-helpers.ts

```diff
@@ -67,5 +72,19 @@ export function createRollupBundle(config: BundleConfig): Promise<any> {
     sourceMap: true
   };
 
+  // When creating a UMD, we want to exclude tslib from the `external` and `globals` bundle options
+  // so that it is inlined into the bundle.
+  if (config.format === 'umd') {
+    bundleOptions.plugins = [rollupNodeResolutionPlugin()];
```

Comment:
I'd rather leave it where it's needed rather than just installing it all the time